client/my-sites/email/email-existing-forwards-notice/index.jsx

```diff
@@ -20,8 +20,7 @@ const EmailExistingForwardsNotice = ( { domainsWithForwards, selectedDomainName
 	return (
 		<Notice showDismiss={ false } status="is-warning">
 			{ translate(
-				"Please note that your existing email forwards for '%(domainName)s' will be removed if you upgrade to a hosted email solution. " +
-					'Use the form below to set up the email addresses you want to continue using.',
+				'Existing email forwards will be removed once you upgrade. Use the form below to set up the email addresses you want to continue using.',
```

Comment:
_Non blocking nit_: Seeing that there could be "technically" multiple forms below (i.e. Google Workspace or Professional Email), do we want to be unequivocal in how we refer to the "form"? 

Am I right in thinking that the user could assume that the currently open form is what we ONLY refer to?

What about:
> Existing email forwards will be removed once you upgrade. Set up the email addresses you want to continue using below ?

I could be spiralling but you can weigh in here @saygunnyc, @semihakocer